Makefile

```diff
@@ -67,11 +67,11 @@ install-kubebuilder:
 
 # Install CRDs into a cluster
 install: manifests
-	kustomize build config/crd | kubectl apply -f -
+	bin/kustomize build config/crd | kubectl apply -f -
```

Comment:
Could you please add an item to your TODO list: pin tools to a specific version

Reply:
They are indeed pinned to a specific version: https://github.com/banzaicloud/istio-operator/blob/v2/Makefile#L10
Or do you mean something else?

Reply:
Sorry, this is perfect! I should've checked the install scripts...